src/modules/md/md.cpp

```diff
@@ -48,6 +49,17 @@ MDModule::MDModule() : Module("MD")
         "Only forces arising from intramolecular terms (including pair potential contributions) will be calculated",
         intramolecularForcesOnly_);
 
+    // Output
+    keywords_.add<OptionalIntegerKeyword>("Output", "EnergyFrequency",
```

Comment:
Sorry - my merging in of #1308 means that your definitions here are in the old format! They should now be like:
```suggestion
    keywords_.add<OptionalIntegerKeyword>("EnergyFrequency",
```